### DISCUSSION TITLE
Heirarchical list of scripts breaks ctrl+f

### DISCUSSION THREAD

**Discussion by xijio** [tteck/Proxmox#670] [Ideas]
The collapsed list of areas makes it hard to find out of something I'm looking for exists.  An easy solution would be expand all button at the top so I can control+f and see if the script I'm looking for exists.

**Comment by tteck** [comment]
Github's markdown is very limited.

**Comment by mkaatman** [comment]
Working on it!

**Comment by tteck** [comment]
@xijio take a look at all of the hard work @mkaatman did on the https://tteck.github.io/Proxmox/ page! All props goes to @mkaatman for getting this going!

**Reply by xijio** [reply]
Wow this is awesome, thanks so much for your hard work, @mkaatman !!
